### ISSUE TITLE
Get logits for pretrained models

### ISSUE THREAD

**Issue by wielandbrendel** [dmlc/mxnet#7105]
Is there any way to get logits (i.e. the activations before the softmax layer) from the pretrained models?

Thanks!

**Comment by thirdwing** [comment]
The following link is an example to extract the pooling results. I hope it helps.

https://github.com/dmlc/mxnet-notebooks/blob/master/python/moved-from-mxnet/predict-with-pretrained-model.ipynb

**Comment by wielandbrendel** [comment]
Thanks @thirdwing, that helped a lot!
